packages/engine/src/worker/runner/javascript/mod.rs

```diff
@@ -1943,7 +1943,17 @@ fn run_experiment(
             v8::V8::initialize_platform(platform);
             v8::V8::initialize();
 
-            let mut isolate = v8::Isolate::new(Default::default());
+            let create_params = v8::Isolate::create_params().heap_limits(
+                init_msg.v8_initial_heap_constraint,
+                init_msg.v8_max_heap_constraint,
+            );
+            let mut isolate = v8::Isolate::new(create_params);
+
+            isolate.add_near_heap_limit_callback(
+                near_heap_limit_callback,
+                // This pointer is not used.
+                (&mut 0 as *mut i32).cast::<std::ffi::c_void>(),
```

Comment:
[`null_mut()`](https://doc.rust-lang.org/std/ptr/fn.null_mut.html)?

Reply:
Oh it was in `std::ptr`... makes sense.
I knew something like this existed, I looked into ffi to find a direct null `*mut c_void`.

Reply:
Addressed in https://github.com/hashintel/hash/pull/446/commits/a4acc363e049958266ce08d52f60f5006b7492cf